src/main/java/net/masterthought/cucumber/json/deserializers/EmbeddingDeserializer.java

```diff
@@ -14,19 +14,36 @@
 import net.masterthought.cucumber.ValidationException;
 import net.masterthought.cucumber.json.Embedding;
 
+import static java.nio.charset.StandardCharsets.UTF_8;
+
 /**
  * Deserializes embedding and stores it in attachment directory.
- * 
+ *
  * @author Damian Szczepanik (damianszczepanik@github)
  */
 public class EmbeddingDeserializer extends CucumberJsonDeserializer<Embedding> {
 
+    /*
+    Base64 Regex matches for the following:
+    ^([A-Za-z0-9+/]{4})* matches group of four chracters from [A-Z, a-z, 0-9, and + /]
+    If a base64 encoded String is not a multiple of four it is padded with '='.
+    (?:[A-Za-z0-9+/]{2}==|[A-Za-z0-9+/]{3}=)?$ checks if the String ends with a group of four of the legal characters
+    or it ends with '=' or '=='
+     */
+    private static final String BASE64_MATCHER_REGEX = "^(?:[A-Za-z0-9+/]{4})*(?:[A-Za-z0-9+/]{2}==|[A-Za-z0-9+/]{3}=)?$";
```

Comment:
Refer to official documentation, API or library so this is clear that this pattern is complete

Reply:
do you mean something like this:
`For details about legal characters and padding refer to https://en.wikipedia.org/wiki/Base64`

Reply:
Nope, rather
https://docs.oracle.com/javase/8/docs/api/java/util/Base64.Decoder.html#decode-java.lang.String-
https://commons.apache.org/proper/commons-codec/apidocs/org/apache/commons/codec/binary/Base64.html#isBase64-java.lang.String-

Reply:
these checks are less specific then the regex.
these checks would consider the String `This is an normal String` as base64 Encoded. Which is obviously wrong.

Reply:
OK, my concern is that I don't want to have own code which decides if this is base64 or not. If that happens I will be happy to merge.

Reply:
Do you realy rather want a lib to be Included to the Project then using a static RegEx?

Reply:
maybe it is already included or maybe you need to prove you implementation is complete

Reply:
See added Unite Test